experimenter/.coveragerc

```diff
@@ -8,6 +8,7 @@ omit =
 [report]
 fail_under = 100
 omit =
+    bin/manifest-tool.py
```

Comment:
Are we still gonna skip coverage checking this whole file?

Reply:
Oh I see, are we invoking the test below outside of the full Experimenter suite?

Reply:
All bin/manifest-tool.py does is call main(). We test the cli in test_cli.py